libsplinter/src/network/connection_manager/auth/mod.rs

```diff
@@ -11,3 +11,5 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+pub mod protocol;
```

Comment:
This should probably go in crate::protocol and follow Transact's lead.

Reply:
Moved.